pkgs/games/tome2/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchFromGitHub,  makeDesktopItem, ncurses, libX11, boost, cmake }:
```

Comment:
Please be consistent with spacing and omit the extra space between `fetchFromGitHub` and `makeDesktopItem`.